provider/overrides/resources.rb

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google Inc.
+# Copyright 2017 Google Inc.
```

Comment:
why this change?

Reply:
good call. I've got the working version in another branch and it's been easier to copy-paste rather than git to get things updated. my bad!